### DISCUSSION TITLE
[Board]: Adafruit Feather RP2040 with RFM95 LoRa Radio

### DISCUSSION THREAD

**Discussion by miltieIV2** [meshtastic/firmware#4405] [Ideas]
### SOC

Other

### Lora IC

SX1272

### Product Link

https://www.adafruit.com/product/5714

### Description

Adafruit Feather RP2040 with RFM95 module.

Created variant `feather_rp2040_rfm95`.
```
extends = rp2040_base
board = adafruit_feather

#define USE_RF95
```

**Comment by miltieIV2** [comment]
This is tested and working, BTW
Config through web client serial, showing up as a node, send and receive messages, etc.

**Comment by miltieIV2** [comment]
This is my first board - what's next?

**Reply by GUVWAF** [reply]
Sounds promising! 
You can open a Pull Request with the required changes.

**Comment by miltieIV2** [comment]
PR: https://github.com/meshtastic/firmware/pull/4451

**Comment by lslewis901** [comment]
Which GPS board if any did you use?

**Reply by miltieIV2** [reply]
I did not attempt to use a GPS board.
I was looking at https://www.adafruit.com/product/4415, but I don't see existing support for the PA1010D  (MTK3333 chipset), but maybe specific support isn't required.

**Reply by fifieldt** [reply]
Generally any UART GPS we can get going :) Support can be added separately to the board.
